Q5.rb

```diff
@@ -0,0 +1,15 @@
+#Now for the previous question, write another method to double all the elements in the array. However, handle edge cases (like array can have a character) as well.
+#Write a method to double all the elements in an array.
+
+def doublearr(double)
```

Comment:
No need to write else cause code will get skipped when `next` is called
also you can use oneliner if

```ruby
def doublearr(double)
  double.each do |x|
    next if ("a".."z") === x
      x *= 2
      print x
  end
end
```